src/tools/chdman.cpp

```diff
@@ -2512,6 +2642,52 @@ static void do_extract_cd(parameters_map &params)
 			// resize the buffer for the track
 			buffer.resize((TEMP_BUFFER_SIZE / output_frame_size) * output_frame_size);
 
+			// GDROM .cue/.bin embeds pregap in output file
+			if (mode == MODE_GDROM_CUEBIN && (trackinfo.pregap > 0) && (trackinfo.pgdatasize > 0))
+			{
+				if ((tracknum == toc.numtrks-1) && (gdrom_identify_pattern(&toc) >= GDROM_TYPE_III))
+				{
+					// special case, 75 frames actual-zeroes, 150 frames data-zeroes with PQ and sync
+					outputoffs += 75 * trackinfo.pgdatasize;
+					trackinfo.frames -= 225;
+					discoffs += 225;
+
+					uint8_t buffer[2352];
```

Comment:
```suggestion
					uint8_t buffer[2352] = {};
```

You have to either initialize the buffer or clear the space in `buffer[2068:2076]` manually for the data to match.

Reply:
Correction: `buffer[2068:2075]`